docker/filetes.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+CLUSTER="devboar"
+
+if [[ "$CLUSTER" != "devboard" ]] && [[ "$CLUSTER" != "dev" ]] && [[ "$CLUSTER" != "devopsdev" ]]
```

Comment:
did you mean "devboar" as above?

This PR MUST be independent of which k8s cluster is ultimately going to deploy a new image